syntax = "proto3";
import "rawTypes.proto";

message Port {
  int32 port = 1;
}

// we must define all inputs/outputs with types defined in protobuf, thus, here.
service DataProcessingService {
  rpc ProcessMeteoData (RawMeteoData) returns (Port) {}
  rpc ProcessPollutionData (RawPollutionData) returns (Port) {}
}